Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/DirectILLCollectData.py

```diff
@@ -68,7 +68,7 @@ def _calibratedIncidentEnergy(detWorkspace, monWorkspace, monEPPWorkspace, eiCal
     instrument = detWorkspace.getInstrument()
     instrument_name = instrument.getName()
     eiWorkspace = None
```

Comment:
That's not really your business, but this declaration is useless because this variable gets another value after, so it could be deleted.